OpenEbooks-Info.plist

```diff
@@ -0,0 +1,89 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!DOCTYPE plist PUBLIC "-//Apple//DTD PLIST 1.0//EN" "http://www.apple.com/DTDs/PropertyList-1.0.dtd">
+<plist version="1.0">
+<dict>
+	<key>CFBundleDevelopmentRegion</key>
+	<string>en</string>
+	<key>CFBundleDisplayName</key>
+	<string>$(PRODUCT_NAME)</string>
+	<key>CFBundleExecutable</key>
+	<string>${EXECUTABLE_NAME}</string>
+	<key>CFBundleIdentifier</key>
+	<string>$(PRODUCT_BUNDLE_IDENTIFIER)</string>
+	<key>CFBundleInfoDictionaryVersion</key>
+	<string>6.0</string>
+	<key>CFBundleName</key>
+	<string>${PRODUCT_NAME}</string>
+	<key>CFBundlePackageType</key>
+	<string>APPL</string>
+	<key>CFBundleShortVersionString</key>
+	<string>$(MARKETING_VERSION)</string>
+	<key>CFBundleSignature</key>
+	<string>????</string>
+	<key>CFBundleSpokenName</key>
+	<string>Open E-Books</string>
+	<key>CFBundleURLTypes</key>
+	<array>
+		<dict>
+			<key>CFBundleTypeRole</key>
+			<string>Editor</string>
+			<key>CFBundleURLName</key>
+			<string>org.nypl.labs.OpenEbooks.clever</string>
+			<key>CFBundleURLSchemes</key>
+			<array>
+				<string>open-ebooks-clever</string>
+			</array>
+		</dict>
+	</array>
+	<key>CFBundleVersion</key>
+	<string>0</string>
+	<key>ITSAppUsesNonExemptEncryption</key>
+	<false/>
+	<key>LSRequiresIPhoneOS</key>
+	<true/>
+	<key>NSAppTransportSecurity</key>
+	<dict>
+		<key>NSAllowsArbitraryLoads</key>
+		<true/>
+	</dict>
+	<key>NSCameraUsageDescription</key>
+	<string>Open eBooks requires access to the camera in order to scan your library card.</string>
+	<key>NSLocationWhenInUseUsageDescription</key>
+	<string>This service is available to New York State residents only. In order to determine your location, please allow access</string>
```

Comment:
Presumably you need to provide a value for this string, but this particular string is inaccurate for Open eBooks. If you do need a value for this string, it would be better to have the equivalent of "this can't happen" so we can treat it as a bug if it ever shows up.

In the medium term we'll hopefully be removing the native card creator code and this can go away when that does.

Reply:
Can confirm they require this string even if we don't actually use location services. As per your suggestion, how about "Oops! Looks like OpenEBooks is accidentally requesting location services. Please report a bug!"

Reply:
Since there's no way to submit a bug within the iOS interface that I can find, I vote "Open eBooks never tracks or uses your location. Please take a screenshot and email it to info@openebooks.net so we can fix this error."